pkg/apis/forklift/v1beta1/plan/vm.go

```diff
@@ -64,7 +64,8 @@ type VMStatus struct {
 	RestorePowerState VMPowerState `json:"restorePowerState,omitempty"`
 	// The firmware type detected from the OVF file produced by virt-v2v.
 	Firmware string `json:"firmware,omitempty"`
-
+	//Choose the primary disk the VM boots from
```

Comment:
this field needs to be the `VM` struct as it's part of the spec of the plan

Reply:
@ahadas I'm not sure I understood what you mean, in the UI we specify it under the  plan once but then @yaacov manually coping it to be under each VM, so we expect to get it in the backend under the VM filed.

Reply:
i think the meaning of it is as i wrote, moving it where LUKS is(VM struct above in the file).
there - we know it is configured by the user (or UI) and won't be edited by the controller. in the status - at least for me it tells me - it might changed by the controller.